tools/python/boutiques/schema/example_bad.json

```diff
@@ -0,0 +1,138 @@
+{
+  "name": "demo-tool",
+  "tool-version": "v1",
+  "description": "a tool that does things",
+  "command-line": "ndmg_pipeline [key1] [key2] [key3] [key4] [key5] [key6] [key7] [flag1]",
+  "container-image": {
+    "type": "docker",
+    "image": "user/image:v1",
+    "index": "http://index.docker.io"
```

Comment:
Tested - successfully caught by JSON validation. I don't want to push this change (ones directly flagged by the schema) to Github in this file as it is designed to pass schema validation but fail extended full Boutiques validation.